test/ToucanRegenBridge.js

```diff
@@ -117,5 +123,14 @@ describe("Bridge contract", function () {
 			const tx = bridge.connect(broker).issueTCO2Tokens(regenUser, broker.address, tco2.address, 100);
 			await expect(tx).to.be.revertedWith("only bridge can issue tokens");
 		});
+
+		it("should mint successfully", async function () {
+			await bridge.connect(broker).bridge(regenUser, tco2.address, 10);
+			expect(await tco2.balanceOf(broker.address)).to.equal(BigNumber.from("1999999999999999999980"));
```

Comment:
Because of the burn that occurs in the test suite above (Polygon to Regen). The TCO2 is the same across all tests but for every test we redeploy the bridge contract. Feel free to change to some other setup that makes sense, not in love with it.

Reply:
ahhh, that's why `totalTransferred` is still only 10. that threw me off because I was suspecting the tco2 is the same across test cases but then thought `totalTransferred` would reflect that. yeah as you say, it's not ideal. especially if new test cases are introduced and put in before this one for one reason or another. I feel we should at least mention this some place so the next person doesn't have to think too long about the implications. 

i currently don't have a good idea how to improve this other than minting new tco2s for at least every test that needs it, but that blows up the duration significantly 🤔

Reply:
> i currently don't have a good idea how to improve this other than minting new tco2s for at least every test that needs it, but that blows up the duration significantly thinking

Exactly, initially I was redeploying the whole infra for every unit test but it was slower than what we currently have.